tests/components/august/test_sensor.py

```diff
@@ -121,3 +125,190 @@
+
+    entity_registry = await hass.helpers.entity_registry.async_get_registry()
+    lock_operator_sensor = entity_registry.async_get(
+        "sensor.online_with_doorsense_name_operator"
+    )
+    assert lock_operator_sensor
+    assert (
+        hass.states.get("sensor.online_with_doorsense_name_operator").state
+        == "Your favorite elven princess"
+    )
+    assert (
+        hass.states.get("sensor.online_with_doorsense_name_operator").attributes[
+            "remote"
+        ]
+        is False
+    )
+    assert (
+        hass.states.get("sensor.online_with_doorsense_name_operator").attributes[
+            "keypad"
+        ]
+        is True
+    )
+    assert (
+        hass.states.get("sensor.online_with_doorsense_name_operator").attributes[
+            "autorelock"
+        ]
+        is False
+    )
+    assert (
+        hass.states.get("sensor.online_with_doorsense_name_operator").attributes[
+            "method"
+        ]
+        == "keypad"
+    )
+
+
+async def test_lock_operator_remote(hass):
+    """Test operation of a lock with doorsense and bridge."""
+    lock_one = await _mock_doorsense_enabled_august_lock_detail(hass)
+
+    activities = await _mock_activities_from_fixture(hass, "get_activity.lock.json")
+    await _create_august_with_devices(hass, [lock_one], activities=activities)
+
+    entity_registry = await hass.helpers.entity_registry.async_get_registry()
+    lock_operator_sensor = entity_registry.async_get(
+        "sensor.online_with_doorsense_name_operator"
+    )
+    assert lock_operator_sensor
+    assert (
+        hass.states.get("sensor.online_with_doorsense_name_operator").state
+        == "Your favorite elven princess"
+    )
+    assert (
+        hass.states.get("sensor.online_with_doorsense_name_operator").attributes[
+            "remote"
+        ]
+        is True
+    )
+    assert (
+        hass.states.get("sensor.online_with_doorsense_name_operator").attributes[
+            "keypad"
+        ]
+        is False
+    )
+    assert (
+        hass.states.get("sensor.online_with_doorsense_name_operator").attributes[
+            "autorelock"
+        ]
+        is False
+    )
+    assert (
+        hass.states.get("sensor.online_with_doorsense_name_operator").attributes[
+            "method"
+        ]
+        == "remote"
+    )
+
+
+async def test_lock_operator_autorelock(hass):
+    """Test operation of a lock with doorsense and bridge."""
+    lock_one = await _mock_doorsense_enabled_august_lock_detail(hass)
+
+    activities = await _mock_activities_from_fixture(
+        hass, "get_activity.lock_from_autorelock.json"
+    )
+    await _create_august_with_devices(hass, [lock_one], activities=activities)
+
+    entity_registry = await hass.helpers.entity_registry.async_get_registry()
+    lock_operator_sensor = entity_registry.async_get(
+        "sensor.online_with_doorsense_name_operator"
+    )
+    assert lock_operator_sensor
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5156575)